### DISCUSSION TITLE
Export and sharing norg files

### DISCUSSION THREAD

**Discussion by ViperX7** [nvim-neorg/neorg#350] [General]
I figured out a way to do this so putting it here to help someone who might be looking for something like this

If someone want to share norg files or export them to other formats without an additional parser,one way to do  that is by using using gnome terminal which provide ability to export terminal output to HTML.

The Final Output compares to original like this 
> yout need to add `<body bgcolor="#282C34">` but thats it
![norg_demo](https://user-images.githubusercontent.com/38191717/151989231-5a60153b-8681-4957-8bba-c20875d5e3dd.png)

It isn't a perfect solution, it's more like a hack but it  gets the job done when I need to share a file with friends

to copy as HTML
- select the text by holding ctrl+shift
- from the menu select edit > copy as HTML
- Paste in a html file and add background color

##### **Limitation** : for files with a lot of lines you will need to 
  1. make the font very small so you can copy a lot of text at one time
  2. or copy different pages one by one
  3. Not sure if this can be automated or not if you find a way to automate this then do let me know
Let me know If someone finds a better way to do it

**Comment by a-berg** [comment]
Given that we have access to a parser, wouldn't it make sense to write some pandoc extension? Just throwing the idea out there.

**Reply by max397574** [reply]
There is a haskell parser in work for pandoc
also there is the exporter module which also allows exporting to other formats
